### ISSUE TITLE
Change `_program-areas` Files From `-card.yml` to `.md`

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#7425]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started](https://www.hackforla.org/getting-started) page and attend an onboarding session.
2. You have already read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
In the `_program-areas` directory, we want to change a file's name and its extension from `-card.yml` to `.md` so that in a future issue
- this file can be used to generate a webpage for that program area automatically
- and the webpage will have the desired URL.

### Details
- On this team, we use collections in Jekyll[^1] to generate some webpages using Markdown files in a collection and an HTML layout. The webpages' URL is based on the name of the Markdown file.
  - For example, a webpage generated from `citizen-engagement.md` will have a URL of `https://www.hackforla.org/program-areas/citizen-engagement` .

### Action Items
- [x] In your IDE, for each file in the `_program-areas` directory[^2], change the name and extension of each file from `-card.yml` to `.md`.
   - [x] `citizen-engagement-card.yml` (e.g., `citizen-engagement.md`)
   - [x] `civic-tech-infrastructure-card.yml`
   - [x] `diversity-equity-inclusion-card.yml`
   - [x] `environment-card.yml`
   - [x] `justice-card.yml`
   - [x] `social-safety-net-card.yml`
   - [x] `vote-representation-card.yml`
   - [x] `workforce-development-card.yml`
- [x] Using Docker, 
   - [x] Check everything has stayed the same for the Program Areas page[^3].
   - [x] Check everything has stayed the same for the Citizen Engagement page[^4]
- [x] Once the PR for this issue is merged, clear the dependency for this issue in https://github.com/hackforla/website/issues/3328
   - [x] Click the checkbox next to this issue number in the dependency list.
      - [x]  If it's the last dependency for the issue and it's in the Icebox column, move to the New Issue Approval column and uncheck the **Dependency label**.
        - <details>
            <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
            <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
          </details>

### Resources/Instructions
[^1]: Info on [Collections in Jekyll](https://jekyllrb.com/docs/collections/)
[^2]: `_program-areas` directory on HfLA's Website repo for QA to check: https://github.com/hackforla/website/tree/gh-pages/_program-areas
[^3]: [Program Areas page](https://www.hackforla.org/program-areas) on live Hack for LA website
[^4]: [Citizen Engagement page](https://www.hackforla.org/citizen-engagement) on live Hack for LA website

**Comment by HackforLABot** [comment]
Hi @JessicaLucindaCheng, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by HackforLABot** [comment]
Hi @JessicaLucindaCheng, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by JessicaLucindaCheng** [comment]
## <a name="title">1. Add a title to the issue</a>

### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
In the `_program-areas` directory, we want to change a file's name and its extension from `-card.yml` to `.md` so that in a future issue
- this file can be used to generate a webpage for that program area automatically
- and the webpage will have the desired URL.

### Details
- On this team, we use collections in Jekyll to generate some webpages using Markdown files in a collection and an HTML layout. The webpages' URL is based on the name of the Markdown file.
  - For example, a webpage generated from `citizen-engagement.md` will have a URL of https://www.hackforla.org/program-areas/citizen-engagement.

### Action Items
- [ ] In your IDE, find `_program-areas/citizen-engagement-card.yml`
- [ ] Change the name and extension of the file from
```
citizen-engagement-card.yml
```
to
```
citizen-engagement.md
```
- [ ] Using Docker, check everything has stayed the same for the [Program Areas page](https://www.hackforla.org/program-areas)

### For Merge Team
- [ ] Once the pull request for this issue is merged, go to #3328 and do the following:
   - [ ] Check off this issue under the _Dependency_ section.
   - [ ] If all the issues in the _Dependency_ section are checked off, move #3328 to the **New Issue Approval** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- Info on [Collections in Jekyll](https://jekyllrb.com/docs/collections/)
- [`_program-areas` directory on HfLA's Website repo](https://github.com/hackforla/website/tree/gh-pages/_program-areas)
- This issue is tracked in https://github.com/hackforla/website/issues/3328.

**Comment by JessicaLucindaCheng** [comment]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
In the `_program-areas` directory, we want to change a file's name and its extension from `-card.yml` to `.md` so that in a future issue
- this file can be used to generate a webpage for that program area automatically
- and the webpage will have the desired URL.

### Details
- On this team, we use [collections in Jekyll](#jekyll-info) to generate some webpages using Markdown files in a collection and an HTML layout. The webpages' URL is based on the name of the Markdown file.
  - For example, a webpage generated from `citizen-engagement.md` will have a URL of https://www.hackforla.org/program-areas/citizen-engagement.

### Action Items
- [ ] In your IDE, under the [`_program-areas` directory](#program-areas-directory), find `citizen-engagement-card.yml`
- [ ] Change the name and extension of the file from
```
citizen-engagement-card.yml
```
to
```
citizen-engagement.md
```
- [ ] Using Docker, check everything has stayed the same for the [Program Areas page](https://www.hackforla.org/program-areas)

### For Merge Team
- [ ] Once the pull request for this issue is merged, go to the [issue(s) dependent on this issue](#issues-dependent-on-this-issue) and do the following:
   - [ ] Check off this issue under the _Dependency_ section.
   - [ ] If all the issues in the _Dependency_ section are checked off, move the issues that have this issue to the **New Issue Approval** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
<ul>
   <li name="jekyll-info">Info on <a href="https://jekyllrb.com/docs/collections/">Collections in Jekyll</a></li>
   <li name="program-areas-directory"><a href="https://github.com/hackforla/website/tree/gh-pages/_program-areas"><i>_program-areas</i> directory on HfLA's Website repo</li>
   <li name="issues-dependent-on-this-issue">Issue(s) dependent on this issue:
   <ul>
      <li>#3328</li>
   </ul>
</ul>

**Comment by JessicaLucindaCheng** [comment]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
In the `_program-areas` directory, we want to change a file's name and its extension from `-card.yml` to `.md` so that in a future issue
- this file can be used to generate a webpage for that program area automatically
- and the webpage will have the desired URL.

### Details
- On this team, we use [collections in Jekyll](#jekyll-info) to generate some webpages using Markdown files in a collection and an HTML layout. The webpages' URL is based on the name of the Markdown file.
  - For example, a webpage generated from `citizen-engagement.md` will have a URL of https://www.hackforla.org/program-areas/citizen-engagement.

### Action Items
- [ ] In your IDE, under the [`_program-areas` directory](#program-areas-directory), find `citizen-engagement-card.yml`
- [ ] Change the name and extension of the file from
```
citizen-engagement-card.yml
```
to
```
citizen-engagement.md
```
- [ ] Using Docker, check everything has stayed the same for the [Program Areas page](https://www.hackforla.org/program-areas)

### For Merge Team
- [ ] Once the pull request for this issue is merged, go to the [issue(s) dependent on this issue](#issues-dependent-on-this-issue) and do the following:
   - [ ] Check off this issue under the _Dependency_ section.
   - [ ] If all the issues in the _Dependency_ section are checked off, move the issues that have this issue to the **New Issue Approval** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
<ul>
   <li name="jekyll-info">Info on <a href="https://jekyllrb.com/docs/collections/">Collections in Jekyll</a></li>
   <li name="program-areas-directory"><a href="https://github.com/hackforla/website/tree/gh-pages/_program-areas"><i>_program-areas</i> directory on HfLA's Website repo</li>
   <li name="issues-dependent-on-this-issue">Issue(s) dependent on this issue:
   <ul>
      <li>#3328</li>
   </ul>
</ul>

**Comment by HackforLABot** [comment]
Hi @caz002, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by caz002** [comment]
i. Availability: 2-8 PM Mondays through Friday
ii. ETA: 6/8/2025
